package/com.unity.formats.usd/Runtime/Scripts/IO/SanitizedSamples.cs

```diff
@@ -9,6 +9,31 @@
 [assembly: InternalsVisibleToAttribute("Unity.Formats.USD.Tests")]
 namespace Unity.Formats.USD
 {
+    /// <summary>
+    /// When a prim is sparsely animated only dynamic properties are read into the Sample.
+    /// The ConversionSate purpose is to hold the data (static properties or computed data) to ensure proper conversion.
+    /// </summary>
+    public interface IRestorable
```

Comment:
Do we need `IRestorable` and `IConversionState`?

Reply:
Because of how the assemblies are separated and used we need an extra layer of abstraction. Without this abstraction we will need to refactor how `USD.NET` and `Unity.Formats.USD` are interacting.